indico/modules/rb_new/client/js/components/TimeRangePicker.jsx

```diff
@@ -25,6 +25,10 @@ import {toMoment, serializeTime} from '../util';
 import './TimeRangePicker.module.scss';
 
 
+const arrowUpKeyCode = 38;
```

Comment:
I'd use all-caps names like `ARROW_UP` and `START_HOUR` here - I think that's more common for this kind of constants (module-level stuff that'll probably never change).

[Interesting post related to it](https://www.reddit.com/r/learnjavascript/comments/5ns7hq/constants_can_be_declared_with_uppercase_or/dce0qdx/)